tests/foreman/ui/test_computeresource.py

```diff
@@ -118,7 +118,7 @@ def test_positive_end_to_end(session, rhev_data, module_org, module_loc, module_
         assert not session.computeresource.search(name)
         assert session.computeresource.search(new_name)[0]['Name'] == new_name
         session.computeresource.delete(new_name)
-        assert not session.computeresource.search(new_name)
+        assert not entities.AbstractComputeResource().search(query={'search': f'name={new_name}'})
```

Comment:
Given that this is a UI test, are there any other ways to assert this query ? Perhaps another use of "session" in the assertion? The same goes for the ones below.

Reply:
The issue is once we delete the entity(compute resource, http proxy etc.) and if there is no other entity of same kind available, we see the main landing page. The `session.computeresource.search(new_name)` kind of assertion won't work because there will be no serchbox. 
There are few other UI test which use same assertion style for this kind of scenario. That's why I changed it this way.
I don't know if there's any other way we can do assertion for this. Any suggestions?

Reply:
That makes sense, should be fine as is.